tests/src/common/Wsk.scala

```diff
@@ -149,9 +149,14 @@ trait ListOrGetFromCollection extends FullyQualifiedNames {
      */
     def get(
         name: String,
+        summary: Boolean = false,
         expectedExitCode: Int = SUCCESS_EXIT)(
             implicit wp: WskProps): RunResult = {
-        cli(wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name)), expectedExitCode)
+        var cliparms = wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name))
+        if (summary) {
+            cliparms = cliparms ++ Seq("--summary")
+        }
+        cli(cliparms, expectedExitCode)
```

Comment:
always consider `var` to be bad. Use it only if you **really** have to. It makes arguing about your code harder.

```
var cliparms = wp.overrides ++ Seq(noun, "get", "--auth", wp.authKey, fqn(name))
if (summary) {
    cli(cliparms ++ Seq("--summary", expectedExitCode)
} else {
    cli(cliparms, expectedExitCode)
}
```

Would be the non-var solution here.